consensus/service/config/src/tokens.rs

```diff
@@ -336,7 +288,7 @@ mod tests {
         assert_eq!(tokens, tokens2);
 
         // Validation should fail since MOB is not specified.
-        assert_validation_error(&tokens, "MOB token configuration not found");
+        assert!(matches!(tokens.validate(), Err(Error::MissingMobConfig)));
```

Comment:
Optional/Equivalent: `assert_matches` prints a better error message
```suggestion
                assert_matches!(data, Err(StreamError::BlockValidation(_))));
```
(requires `use std::assert_matches`)

Reply:
Nice

Reply:
However, it requires `#![feature(assert_matches)]` so I am not sure if this is actually superior.